compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTileAndDistribute.cpp

```diff
@@ -206,25 +206,15 @@ static SmallVector<int64_t> getTransposedOperands(linalg::GenericOp linalgOp) {
 using PromotionFilterFunction = std::function<LogicalResult(Operation *op)>;
 
 /// Returns true if op is appropriate transpose for promotion.
-static LogicalResult transposeFilter(
-    Operation *op, ArrayRef<int64_t> filterOperandsToPromote) {
+static LogicalResult transposeFilter(Operation *op,
+                                     linalg::GenericOp promotedFilterOp) {
   auto linalgOp = dyn_cast<linalg::GenericOp>(op);
   if (!linalgOp) return failure();
 
   if (linalgOp.getNumParallelLoops() < 2) {
```

Comment:
This needs to go in `getTransposedOperands`? It sounds like the code line 195 assumes at least two loops